doc/level1.rst

```diff
@@ -710,6 +701,79 @@ way:
 4. Set ``peers[gossip-addr].gossip_key`` to the value of the
    ``keydata`` attribute.
 
+E-Mail Drafts
+-------------
+
+During message composition, the user may want to close the dialog and
+resume composition at a later point.  To preserve the session state,
+messages are typically saved as "Drafts" in a special folder.
+Messages from this folder may also be resumed from a different MUA.
+
+To prevent the cleartext of a message draft from leaking to the
+provider, the MUA can encrypt the draft message to itself (but not the
+recipients) before uploading.  It MUST encrypt in this way if the
+message is going to be sent encrypted, and SHOULD encrypt if the user
+might decide to encrypt at a later point, or otherwise keep the draft
+local only.  Drafts encrypted in this way SHOULD be stored in PGP/MIME
+format.
+
+To allow storage of drafts independently from the user's secret key,
+encrypted drafts do not need to be signed.  Conversely, a MUA SHOULD
+ignore the signature status of encrypted drafts.
```

Comment:
This makes it difficult to defend against draft injection by unauthorized clients.  I know that's an active attack and therefore out of scope, but it's not clear what we gain by ignoring this status.

Reply:
An attacker doesn't know the plaintext, so they can at best blindly inject drafts into the user's mailbox - which they can already do in plaintext. And considering the data is encrypted to the user's key, does the signed document for publication part really matter? I tried adding that part in some phrasings just now, but I felt like it made the pargraph unnecessarily wordy.